src/FrameRipper.py

```diff
@@ -0,0 +1,79 @@
+import cv2
+import numpy as np
+import os 
+
+#Frame Ripper Script WIP
+#Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+#
+#Steve
+
+#Function to scan the video files, processes and adds them to Array video_list
+def getVideoFiles():
+    
+    entries = os.listdir('Videos')
+    video_list = []
+    
+    for entry in entries:
+        video_list.append("Videos/" + entry)
+    return video_list
+
+#Function that takes each element from the video_list and returns the total frame count of each video file
+def getTotalFrames():
+    
+    videos = getVideoFiles()
```

Comment:
Consider passing this in as an argument instead. The function would then be more general, not coupled with the above function. Then just call `get_total_frames(get_video_files())`.